docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  neo4j:
+    environment:
+      # The initial neo4j username MUST BE "neo4j" if a db user has not been previously created.
+      # The initial neo4j passworrd is tied to the docker volume
+      # If you want to update the password you must update the account in neo4j
+      # Then, you need to relaunch cartography process so it has the updated password
+      - NEO4J_AUTH=${NEO4J_USERNAME}/${NEO4J_PASSWORD}
+    image: neo4j:3.5
+    healthcheck:
+      test: curl -i http://127.0.0.1:7474 2>&1 | grep -c -e '200 OK'
+    ports:
+      - "7687:7687" # host:container
+      - "7474:7474"
+    volumes:
+      - $HOME/neo4j/data:/data
+
+  cartography:
+    environment:
```

Comment:
We might want to think how to add support for other providers currently supported by Cartography (GCP, GSuite, Okta...)

Reply:
We can update the shell file that checks for the existence of those env vars and then runs cartography with the appropriate shell commands.

I figured doing this as the MVP and then getting support for everything else next will be much easier by me or anyone else needing those features.

Reply:
Yes, that is the goal. There may need to be some updates to the shell script to test for those env vars and apply the correct flags when running cartography